cypress/integration/components/AsyncSelect.spec.js

```diff
@@ -0,0 +1,48 @@
+describe("AsyncSelect", () => {
+  const getSelectComponent = () => cy.get("[data-testid='select-container']");
+  const getDropdownMenu = () => cy.get("[data-testid='select-dropdown']");
+  const getMultiselect = () => getSelectComponent();
+  const getClearButton = () => cy.get("[data-testid='select-clear']");
+  const assertDropDownIsOpen = () => getDropdownMenu().should("exist");
+
+  describe("Multiselect", () => {
+    beforeEach(() => {
+      cy.renderFromStorybook("asyncselect--multiselect");
+    });
+
+    it("can select multiple values", () => {
+      getMultiselect().click();
+
+      cy.focused().type("cana");
+      assertDropDownIsOpen().contains("Canada");
+      cy.focused().type("{enter}");
+
+      cy.focused().type("austra");
+      assertDropDownIsOpen().contains("Australia");
+      cy.focused().type("{enter}");
+
+      getMultiselect().contains("Canada");
+      getMultiselect().contains("Australia");
+    });
+
+    it("clears all selected values", () => {
+      getMultiselect().click();
+
+      cy.focused().type("cana");
+      assertDropDownIsOpen().contains("Canada");
+      cy.focused().type("{enter}");
+
+      getClearButton().click();
+
+      getMultiselect().contains("Please select a countries");
+    });
+
+    it("does not show the dropdown arrow", () => {
+      getDropdownMenu().should("not.exist");
```

Comment:
Is `getDropDownMenu` what we're trying to test here or should we have a data-test id on the arrow itself to just check if it's showing?

Reply:
You're right, we're not asserting the right thing.

Reply:
I added a new test that asserts that the dropdown arrow is not showing using its data-testid.